quickstarts/coredns/config.yml

```
@@ -0,0 +1,50 @@
+# Name of the quickstart (required)
+slug: coredns-metrics
```

In order for us to not have to manually delete the previous quickstart in all different environments in the database, we need to add back the previous id from the deleted config file, along with the same slug.

After discussing with @seemantk - we want to ensure we are not _deleting_ any quickstarts. Therefore, an updated quickstart is required here.

By bringing back the same `id` and `slug`, it will update the previous quickstart which is okay in this scenario.

We also do not want to change the URL already established, so the slug must be the same as the previous quickstart.

```suggestion
id: c9ac4abb-4a61-4eee-9e1e-79506806684e
slug: coredns
```